model/cluster_utility.go

```diff
@@ -31,6 +31,10 @@ const (
 	TeleportDefaultVersion = "0.3.0"
 )
 
+// UtilityValuesDirectory is the directory in which Helm charts and
+// other Utility values are stored
+var UtilityValuesDirectory string
```

Comment:
I am a tiny bit wary about using a shared variables for this, but it should be fine. Perhaps we can have a `Set` method for it and not export it to help ensure we don't update it when we shouldn't? It would also be nice if when retrieving the value we had method like `GetUtilityValues(yaml string)` that would create the complete directory string for you so there was less chance of error there.

Reply:
Sure these are both reasonable improvements to the PR as written, can do.